src/modules/event/event.h

```diff
@@ -1,5 +1,6 @@
 #include <stdbool.h>
 #include <stdint.h>
+#include "core/os.h"
```

Comment:
strongly prefer to omit this include from header, not sure how to handle that though, also nit trailing comma below